### PR TITLE
Fancier design & add single stream to calendar

### PR DESCRIPTION
**Pull request by Larsklopstra**
This PR introduces:

- Fancier design
- Add single stream to calendar rather than all
- Cleaner blade formatting

### CODE REVIEW

**Comment by christophrumpel** [comment]
Thanks @Larsklopstra, the update looks really nice 🔥 Just two little additions:

- please make the title a bit bigger again, maybe default 5xl and 4xl on smaller screen and space-y  then maybe 4
- Can you bring in the latest changes from the main branch, I think it will be easier for you to merge it
- @freekmurze please take a look at the single cal events integration

**Comment by Larsklopstra** [comment]
Updated! @christophrumpel

File: app/Http/Controllers/AddSingleStreamToCalendarController.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+use App\Models\Stream;
+use Spatie\IcalendarGenerator\Components\Calendar;
+
+class AddSingleStreamToCalendarController extends Controller
+{
+    public function __invoke(Stream $stream)
+    {
+        $calendar = Calendar::create()
+            ->name("Larastreamers: watch {$stream->title}")
+            ->description('There is no better way to learn than by watching other developers code live. Find out who is streaming next in the Laravel world.');
```
Comment (christophrumpel) [inline app/Http/Controllers/AddSingleStreamToCalendarController.php:14]: Just noticed that we do not store the description of the stream. Later we should change this.

File: database/seeders/TestDataSeeder.php [hunk 2]
```diff
@@ -19,5 +21,13 @@ public function run()
             'email' => 'test@test.at',
             'password' => bcrypt('test'),
         ]);
+
+        Channel::factory()
```
Comment (christophrumpel) [inline database/seeders/TestDataSeeder.php:25]: Are you using this new data somewhere?

- I believe this file is a left over which is not used (I need to delete it later)
- I am missing a test for single cal download. Can you add that?

**Review by christophrumpel — CHANGES_REQUESTED** [review]
Just looked through it again and added notes. Please give it a look. It would have been a little easier if it was were two PRs, then I could already merge the fancy new design :-)

File: database/seeders/TestDataSeeder.php (hunk 2 above)
Comment (Larsklopstra) [reply database/seeders/TestDataSeeder.php:25]: Sure will do

**Comment by Larsklopstra** [comment]
@christophrumpel Removed the factory and added the test

**Comment by christophrumpel** [comment]
Thanks @Larsklopstra. Look good. Will merge it now because I do not want to wait till GitHub's actions are back. 👍

**Pull request merged**